CHANGELOG.md

```diff
@@ -10,6 +14,30 @@
 * Support named backlinks on realm models. You can now add and annotate a realm object iterator field with `@Backlink(#fieldName)`. ([#996](https://github.com/realm/realm-dart/pull/996))
 * Allow `@Indexed` attribute on all indexable type, and ensure appropriate indexes are created in the realm. ([#797](https://github.com/realm/realm-dart/issues/797))
 * Add `parent` getter on embedded objects. ([#979](https://github.com/realm/realm-dart/pull/979))
+* Support [Client Resets](https://www.mongodb.com/docs/atlas/app-services/sync/error-handling/client-resets/). Flutter SDK automatically detects the need for client resets and automatically performs it according to the configured callbacks for the type of client reset handlers set to `FlexibleSyncConfiguration`. 
+  Added a parameter `clientResetHandler` to `Configuration.flexibleSync`. 
+  Supported client reset handlers are `ManualRecoveryHandler`, `DiscardUnsyncedChangesHandler`, `RecoverUnsyncedChangesHandler` and `RecoverOrDiscardUnsyncedChangesHandler`.
+ `RecoverOrDiscardUnsyncedChangesHandler` is the default strategy. An example usage of the default `clientResetHandler` is as follows:
+  ```dart
+      final config = Configuration.flexibleSync(user, [Task.schema],
+        clientResetHandler: RecoverOrDiscardUnsyncedChangesHandler(
+          // The following callbacks are optional.
+          onBeforeReset: (beforeResetRealm) {
+            // Executed right before a client reset is about to happen.
+            // If an exception is thrown here the recovery and discard callbacks are not called.
+          },
+          onAfterRecovery: (beforeResetRealm, afterResetRealm) {
+            // Executed right after an automatic recovery from a client reset has completed.
+          },
+          onAfterDiscard: (beforeResetRealm, afterResetRealm) {
+            // Executed after an automatic recovery from a client reset has failed but the Discard has completed.
+          },
+          onManualResetFallback: (clientResetError) {
+            // Handle the reset manually in case some of the callbacks above throws an exception
+          },
+        )
+    );
+```
```

Comment:
I miss a ref to an issue/pr here

Reply:
its the `Support Client Reset` bullet above